File: archives/leetcode2/0190. Reverse Bits.py
class Solution:
    # @param n, an integer
    # @return an integer
    def reverseBits(self, n):
        if n==0:
            return 0
        res=0
        for _ in range(32):
            res=res<<1
            res=res+(n&1)
            n=n>>1    
        return res

File: archives/C#/0190. Reverse Bits.cs
public class Solution {
    public uint reverseBits(uint n) {
        uint rep=0;
        for(int i=0;i<32;i++){
            uint tmp=(n&1);
            rep=(rep<<1)+tmp;
            n=n>>1;
        }
        return rep;
    }
}
